pages/sortinglist/[slug].js

```diff
@@ -117,51 +117,54 @@ export default function DynamicForm() {
   const currentTree = treeData.find((element) => element.slug === treeFromSlug);
   const attributes = currentTree.attributes;
 
+  let inputNumber = 1;
+
+  const renderInputField = (label) => {
+    const inputId = `input$(inputNumber)`;
+    inputNumber++;
+    return (
+      <p>
+        <label htmlFor={inputId}>{label}</label>
+        <input type="text" name={inputId} id={inputId} />
+      </p>
+    );
+  };
+
   return (
-    <div>
-      <Heading>Formular</Heading>
+    <>
+      <Heading>Sortierung</Heading>
       <StyledLink href="/treelist">← zurück</StyledLink>
       <StyledForm>
-        {attributes.auswuechse && <div>Auswüchse</div>}
-
-        {attributes.aeste && <div>Äste????</div>}
-        {attributes.wasserreiser && <div>Wasserreiser</div>}
-
-        {attributes.drehwuchs && <div>Drehwuchs (cm)</div>}
-        {attributes.mondring && <div>Mondring ?</div>}
-        {attributes.einfachekrümmung && <div>Einfache Krümmung ?</div>}
-
-        {attributes.sternriss && <div>Sternriss ?</div>}
-
-        {attributes.frostriss && <div>Frostriss ?</div>}
-        {attributes.ringrisse && <div>Ringrisse- / schäle?</div>}
-        {attributes.insektenfraßgänge && <div>Insektenfraßgänge ?</div>}
-        {attributes.fäule && <div>Fäule ?</div>}
+        {attributes.auswuechse && renderInputField("Auswüchse ?")}
```

Comment:
```suggestion
        {attributes.auswuechse && <InputField label="Auswüchse ?"/> }
```

This could be a component, with the label prop.